Add a convenient constructor

larryliu0820 opened the pull request:
Summary: As titled. Use default values for `topp` and `rng`.

Differential Revision: D71956172

swolchok commented on extension/llm/runner/stats.h:
```diff
@@ -82,8 +82,6 @@ struct ET_EXPERIMENTAL Stats {
   long aggregate_sampling_timer_start_timestamp = 0;
 };
 
-static constexpr auto kTopp = 0.9f;
```
isn't this bc-breaking because this header doesn't include sampler.h?

larryliu0820 replied:
This constant belong to `sampler.h` not `stats.h`. Anyone using this constant should already imported `sampler.h` so it's not breaking anything. They don't have to include `stats.h` right now if they just need to use this constant.

larryliu0820 committed:
Add a convenient constructor (#9707)

Summary:

As titled. Use default values for `topp` and `rng`.

Reviewed By: iseeyuan

Differential Revision: D71956172
